### DISCUSSION TITLE
Could not find the qt platform plugin "wayland" in ""

### DISCUSSION THREAD

**Discussion by enrico223** [ahrm/sioyek#788] [Q&A]
Since I had a issue with the archlinux package [#786](https://github.com/ahrm/sioyek/issues/786#issue-1843101753), I installed the appimage, but I get the following error when starting it on Hyprland
```
qt.qpa.plugin: Could not find the Qt platform plugin "wayland" in ""
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: xcb.

Aborted (core dumped)
```
I tried reinstalling some qt packages, but no progress. 
The appimage works on GNOME.

**Comment by ahrm** [comment]
I am not sure about this. Does this help? : https://unix.stackexchange.com/questions/598099/could-not-find-the-qt-platform-plugin-wayland

**Reply by enrico223** [reply]
Setting `QT_QPA_PLATFORM=wayland;xcb` make it work, but since I have hidpi display and fractional scaling, it looks blurry/unusable. 
The `exec` section in `sioyek.desktop` has this envs: `DESKTOPINTEGRATION=0 APPIMAGELAUNCHER_DISABLE=1`, what do these vars do? Could they be the problem? 
Should I file a bug?

**Reply by ahrm** [reply]
I don't think those variables are the problem. We already have an open issue about blurry font.
Does the blurry font issue happen on gnome too without `QT_QPA_PLATFORM=wayland;xcb`?

**Reply by enrico223** [reply]
On GNOME I have no problem without `QT_QPA_PLATFORM` var set, it works out-of-the-box.

**Reply by ahrm** [reply]
I guess the dpi reported is wrong on wayland. I assume it is either a wayland or a qt bug. I don't think it is a sioyek bug.

**Comment by DedZago** [comment]
This is still an issue for me on Hyprland, has anyone found a workaround?
